### ISSUE TITLE
0.3.4 build fails on El Capitan

### ISSUE THREAD

**Issue by ilovezfs** [halo/macosvpn#37]
```
==> xcodebuild SYMROOT=build
Build settings from command line:
    SYMROOT = build

2018-01-01 04:19:18.139 xcodebuild[3276:10743] Failed to locate a valid instance of CoreSimulatorService in the bootstrap.  Adding it now.
2018-01-01 04:19:18.140 xcodebuild[3276:10743] Failed to load CoreSimulatorService.  Access to simulator services will not be available.  Error=Error Domain=NSPOSIXErrorDomain Code=61 "Connection refused" UserInfo={NSLocalizedDescription=Unable to lookup com.apple.CoreSimulator.CoreSimulatorService in the bootstrap.  This can happen if running with a sandbox profile.  When running with a sandbox profile, /Applications/Xcode.app/Contents/Developer/Library/PrivateFrameworks/CoreSimulator.framework/XPCServices/com.apple.CoreSimulator.CoreSimulatorService.xpc must be owned by root, not group writable, and not world writable.  See <rdar://problem/22142915>.}
2018-01-01 04:19:18.140 xcodebuild[3276:10746] CoreSimulatorService connection became invalid.  Simulator services will no longer be available.
2018-01-01 04:19:18.141 xcodebuild[3276:10743]  DVTAssertions: Warning in /Library/Caches/com.apple.xbs/Sources/DVTiOSFrameworks/DVTiOSFrameworks-11753/IDEiOSSupportCore/DVTiPhoneSimulatorLocator.m:105
Details:  [SimServiceContext defaultDeviceSetWithError:] returned nil (Error Domain=NSPOSIXErrorDomain Code=53 "Software caused connection abort" UserInfo={NSLocalizedFailureReason=Failed to subscribe to notifications from CoreSimulatorService., NSLocalizedDescription=Failed to initialize simulator device set., NSUnderlyingError=0x7ffd4624b110 {Error Domain=NSPOSIXErrorDomain Code=53 "Software caused connection abort" UserInfo={NSLocalizedDescription=Error returned in reply to notification request: Connection invalid}}}). Simulator device support disabled.
Object:   <DVTiPhoneSimulatorLocator: 0x7ffd43e7e9c0>
Method:   -startLocating
Thread:   <NSThread: 0x7ffd41518a40>{number = 3, name = (null)}
Please file a bug at http://bugreport.apple.com with this warning message and any useful information you can provide.
2018-01-01 04:19:18.142 xcodebuild[3276:10745] Failed to locate a valid instance of CoreSimulatorService in the bootstrap.  Adding it now.
2018-01-01 04:19:18.143 xcodebuild[3276:10745] Failed to load CoreSimulatorService.  Access to simulator services will not be available.  Error=Error Domain=NSPOSIXErrorDomain Code=61 "Connection refused" UserInfo={NSLocalizedDescription=Unable to lookup com.apple.CoreSimulator.CoreSimulatorService in the bootstrap.  This can happen if running with a sandbox profile.  When running with a sandbox profile, /Applications/Xcode.app/Contents/Developer/Library/PrivateFrameworks/CoreSimulator.framework/XPCServices/com.apple.CoreSimulator.CoreSimulatorService.xpc must be owned by root, not group writable, and not world writable.  See <rdar://problem/22142915>.}
2018-01-01 04:19:18.143 xcodebuild[3276:10738] CoreSimulatorService connection became invalid.  Simulator services will no longer be available.
=== BUILD TARGET macosvpn OF PROJECT macosvpn WITH THE DEFAULT CONFIGURATION (Release) ===

Check dependencies

Write auxiliary files
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/swift-overrides.hmap
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-own-target-headers.hmap
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn.hmap
/bin/mkdir -p /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/macosvpn.LinkFileList
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/macosvpn-OutputFileMap.json
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-all-non-framework-target-headers.hmap
/bin/mkdir -p /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/SharedPrecompiledHeaders/macosvpn-Prefix-ewfsgrnshsialocyemafavpvdusc
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/SharedPrecompiledHeaders/macosvpn-Prefix-ewfsgrnshsialocyemafavpvdusc/macosvpn-Prefix.pch.pch.hash-criteria
/bin/mkdir -p /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/SharedPrecompiledHeaders/macosvpn-Prefix-aptyfoysyfsgsigbklgmurfbckzk
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/SharedPrecompiledHeaders/macosvpn-Prefix-aptyfoysyfsgsigbklgmurfbckzk/macosvpn-Prefix.pch.pch.hash-criteria
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-generated-files.hmap
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-all-target-headers.hmap
write-file /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-project-headers.hmap

CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
    cd /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4
    export DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer
    export SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk
    export TOOLCHAINS=com.apple.dt.toolchain.XcodeDefault
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -incremental -module-name macosvpn -O -whole-module-optimization -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -target x86_64-apple-macosx10.9 -g -Xfrontend -serialize-debugging-options -I /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/Release -F /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/Release -F /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor -c -num-threads 8 /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/StyleParameter.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/Wrap.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNColor.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNCommand.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNHelp.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/Color\ ·\ Named.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNController.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/Color.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/ECMA\ 48.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNAuthorizations.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNLogger.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/Color\ ·\ EightBit.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNExitCode.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNCommandType.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNCFArray+Sequence.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNLogFormatter.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNServiceCreator.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/Optional+String.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/CocoaLumberjack/CocoaLumberjack.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNServiceType.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNServiceRemover.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/main.swift -output-file-map /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/macosvpn-OutputFileMap.json -parseable-output -serialize-diagnostics -emit-dependencies -emit-module -emit-module-path /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/macosvpn.swiftmodule -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/swift-overrides.hmap -Xcc -iquote -Xcc /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-generated-files.hmap -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-own-target-headers.hmap -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-all-target-headers.hmap -Xcc -iquote -Xcc /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-project-headers.hmap -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/Release/include -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/DerivedSources/x86_64 -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/DerivedSources -emit-objc-header -emit-objc-header-path /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/macosvpn-Swift.h -import-objc-header /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/macosvpn-Bridging-Header.h -Xcc -working-directory/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4

CompileSwift normal x86_64
    cd /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/StyleParameter.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/Wrap.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNColor.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNCommand.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNHelp.swift "/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/Color · Named.swift" /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNController.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/Color.swift "/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/ECMA 48.swift" /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNAuthorizations.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNLogger.swift "/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/PrettyColors/Color · EightBit.swift" /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNExitCode.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNCommandType.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNCFArray+Sequence.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNLogFormatter.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNServiceCreator.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/Optional+String.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor/CocoaLumberjack/CocoaLumberjack.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNServiceType.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNServiceRemover.swift /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/main.swift -target x86_64-apple-macosx10.9 -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -I /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/Release -F /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/Release -F /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Vendor -g -import-objc-header /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/macosvpn-Bridging-Header.h -serialize-debugging-options -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/swift-overrides.hmap -Xcc -iquote -Xcc /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-generated-files.hmap -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-own-target-headers.hmap -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-all-target-headers.hmap -Xcc -iquote -Xcc /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/macosvpn-project-headers.hmap -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/Release/include -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/DerivedSources/x86_64 -Xcc -I/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/DerivedSources -Xcc -working-directory/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4 -emit-module-doc-path /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/macosvpn.swiftdoc -O -module-name macosvpn -emit-module-path /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/macosvpn.swiftmodule -emit-objc-header-path /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/macosvpn-Swift.h -serialize-diagnostics-path /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/StyleParameter.dia -emit-dependencies-path /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/StyleParameter.d -num-threads 8 -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/StyleParameter.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/Wrap.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNColor.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNCommand.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNHelp.o -o "/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/Color · Named.o" -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNController.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/Color.o -o "/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/ECMA 48.o" -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNAuthorizations.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNLogger.o -o "/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/Color · EightBit.o" -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNExitCode.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNCommandType.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNCFArray+Sequence.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNLogFormatter.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNServiceCreator.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/Optional+String.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/CocoaLumberjack.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNServiceType.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/VPNServiceRemover.o -o /tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/build/macosvpn.build/Release/macosvpn.build/Objects-normal/x86_64/main.o
/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNController.swift:46:31: error: binary operator '==' cannot be applied to operands of type 'UInt' and 'UInt8'
    if VPNArguments.command() == VPNCommandType.Create {
       ~~~~~~~~~~~~~~~~~~~~~~ ^  ~~~~~~~~~~~~~~~~~~~~~
/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNController.swift:46:31: note: overloads for '==' exist with these partially matching parameter lists: (UInt8, UInt8), (UInt, UInt)
    if VPNArguments.command() == VPNCommandType.Create {
                              ^
/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNController.swift:50:36: error: binary operator '==' cannot be applied to operands of type 'UInt' and 'UInt8'
    else if VPNArguments.command() == VPNCommandType.Delete {
            ~~~~~~~~~~~~~~~~~~~~~~ ^  ~~~~~~~~~~~~~~~~~~~~~
/tmp/macosvpn-20180101-3267-15hlo4d/macosvpn-0.3.4/macosvpn/Classes/VPNController.swift:50:36: note: overloads for '==' exist with these partially matching parameter lists: (UInt8, UInt8), (UInt, UInt)
    else if VPNArguments.command() == VPNCommandType.Delete {
                                   ^

** BUILD FAILED **


The following build commands failed:
	CompileSwift normal x86_64
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
```

**Comment by halo** [comment]
Alas, which version of Xcode by the way?

**Comment by ilovezfs** [comment]
```
iMac-TMP:~ joe$ brew config
HOMEBREW_VERSION: 1.4.1-49-g4e08da9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 4e08da95c34c27ba6d5ca5f80718fe2788337cdf
Last commit: 5 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: b7ad96b6705b46fb314aa7b9e33ab42e60d30983
Core tap last commit: 25 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_DEVELOPER: 1
HOMEBREW_FORCE_VENDOR_RUBY: 1
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MERGE: 1
HOMEBREW_NO_ANALYTICS: 1
HOMEBREW_NO_AUTO_UPDATE: 1
HOMEBREW_SANDBOX: 1
CPU: octa-core 64-bit skylake
Homebrew Ruby: 2.3.3 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.3/bin/ruby
Clang: 8.0 build 800
Git: 2.10.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.43.0 => /usr/bin/curl
Perl: /usr/bin/perl
Python: /usr/local/opt/python/libexec/bin/python => /usr/local/Cellar/python@2/2.7.14_1/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 9.0.1, 1.8.0_152, 1.8.0_151, 1.7.0_154-zulu-7.20.0.3
macOS: 10.11.6-x86_64
Xcode: 8.2.1
CLT: 8.2.0.0.1.1480973914
X11: 2.7.11 => /opt/X11
iMac-TMP:~ joe$ 
```

**Comment by halo** [comment]
I don't have Xcode 8.2.1 or 8.3 available at the moment, but I think I fixed the issue driving blindly :)

I'm waiting for Travis [to build this](https://travis-ci.org/halo/macosvpn/jobs/324155927) on Xcode 8.3 before I publish a new release.

If you want, you may gladly try the master branch. If it builds, I'll ship it without waiting for Travis.

Thank you!

**Comment by halo** [comment]
Ok, it [compiles fine](https://travis-ci.org/halo/macosvpn/jobs/324504789/config) using Xcode 8.3 on Travis after the fix. I'll keep Travis testing multiple Xcode versions now so that this doesn't happen again.

I released macosvpn [`0.3.5`](https://github.com/halo/macosvpn/releases/tag/0.3.5) accordingly.

Thank you for reporting!

**Comment by ilovezfs** [comment]
Thanks for the fix @halo! Voilà https://github.com/Homebrew/homebrew-core/pull/22407
